src/components/Landing.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+const Login = () => (
```

Comment:
The component name does not match the name of the file.

Reply:
@adesege worked on feedback